### ISSUE TITLE
❌ Error 404 en salud del sitio

### ISSUE THREAD

**Issue by PartidodelTrabajo** [elementor/elementor#20722]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Al querer ingresar al Estado de Salud del Sitio, me aparece el error 404

### Steps to reproduce

1. Al querer ingresar al Estado de Salud del Sitio, me aparece el error 404

### Isolating the problem

- [ ] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MySQL Community Server - (GPL) v5.7.40
	PHP Version: 8.1.13
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://ptjalisco.org
	Home URL: https://ptjalisco.org
	WP Multisite: No
	Max Upload Size: 8 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /%category%/%postname%/
	Language: es
	Timezone: America/Mexico_City
	Admin Email: info@ptjalisco.org
	Debug Mode: Inactive

== Theme ==
	Name: Polity Lite
	Version: 2.0
	Author: Grace Themes
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: es_ES
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/108.0.0.0 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Complianz - Terms and Conditions
		Version: 1.1.3
		Author: Really Simple Plugins

	Complianz | GDPR/CCPA Cookie Consent
		Version: 6.3.5
		Author: Really Simple Plugins

	Elementor
		Version: 3.10.0-dev1
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Limit Login Attempts Reloaded
		Version: 2.25.10
		Author: Limit Login Attempts Reloaded

	Really Simple SSL
		Version: 6.0.12
		Author: Really Simple Plugins

	WP File Manager
		Version: 7.1.7
		Author: mndpsingh287

== Must-Use Plugins ==
	WordPress automation by Installatron
		Version: 
		Author: 

== Experimentos de Elementor ==
	Salida del DOM optimizada: Activo
	Carga de recursos mejorada: Activo
	Carga de CSS mejorada: Activo
	Fuentes de iconos integradas: Activo
	Mejoras de accesibilidad: Activo
	Puntos de ruptura personalizados adicionales: Activo
	Importar o exportar un kit web: Activo
	Ocultar los widgets nativos de WordPress de los resultados de búsqueda: Activo
	admin_menu_rearrangement: Inactivo por defecto
	Contenedor Flexbox: Activo
	Páginas de destino: Activo
	Muestrario de colores: Activo
	Widgets favoritos: Activo
	Barra superior de administración: Activo
	Carga diferida de imágenes de fondo: Activo por defecto
	Guardar como por defecto: Inactivo por defecto

== Registro ==
	
Log: showing 20 of 372022-08-25 15:13:14 [info] Elementor/Upgrades - _on_each_version Start  
2022-08-25 15:13:14 [info] Elementor/Upgrades - _on_each_version Finished 
2022-08-25 15:13:14 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0-dev1',
  'to' => '3.8.0-dev4',
)]
2022-10-07 15:21:57 [info] elementor::elementor_updater Started 
2022-10-07 15:21:57 [info] Elementor/Upgrades - _on_each_version Start  
2022-10-07 15:21:57 [info] Elementor/Upgrades - _on_each_version Finished 
2022-10-07 15:21:57 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Start  
2022-10-07 15:21:57 [info] Elementor/Upgrades - _v_3_8_0_fix_php8_image_custom_size Finished 
2022-10-07 15:21:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0-dev4',
  'to' => '3.9.0-dev2',
)]
2022-10-07 15:21:57 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.8.0-dev4',
  'to' => '3.9.0-dev2',
)]
2022-11-04 11:59:11 [info] elementor::elementor_updater Started 
2022-11-04 11:59:11 [info] Elementor/Upgrades - _on_each_version Start  
2022-11-04 11:59:11 [info] Elementor/Upgrades - _on_each_version Finished 
2022-11-04 11:59:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0-dev2',
  'to' => '3.9.0-dev4',
)]
2022-11-04 11:59:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0-dev2',
  'to' => '3.9.0-dev4',
)]
2022-12-01 16:21:49 [info] elementor::elementor_updater Started 
2022-12-01 16:21:49 [info] Elementor/Upgrades - _on_each_version Start  
2022-12-01 16:21:49 [info] Elementor/Upgrades - _on_each_version Finished 
2022-12-01 16:21:49 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0-dev4',
  'to' => '3.10.0-dev1',
)]
2022-12-01 16:21:49 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.9.0-dev4',
  'to' => '3.10.0-dev1',
)]

PHP: showing 2 of 2PHP: 2022-07-26 12:16:02 [notice X 1][/home/ptjalisc/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-12-09 16:03:49 [notice X 4694][/home/ptjalisc/public_html/wp-content/plugins/elementor/modules/performance-lab/module.php::31] Optional parameter $size declared before required parameter $url is implicitly treated as a required parameter [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 3 of 3JS: 2022-08-02 21:10:15 [error X 8][https://ptjalisco.org/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.8.0-dev1:3:765816] this.store.find(...) is undefined 
JS: 2022-11-04 18:05:12 [error X 22][https://ptjalisco.org/wp-includes/js/jquery/jquery.min.js?ver=3.6.0:2:31703] Cannot read properties of undefined (reading \'tools\') 
JS: 2022-12-09 22:08:14 [error X 27][https://ptjalisco.org/wp-content/plugins/elementor/assets/js/frontend.min.js?ver=3.10.0-dev1:2:32848] elementorFrontendConfig is not defined 

== Elementor - Compatibility Tag ==

**Comment by nicholaszein** [comment]
Hello @PartidodelTrabajo, 

Thank you for submitting your issue! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information.

- **Clear and concise title:** Please provide a clear and concise title that gives context to your issue;
- **Prerequisites:** please search the repository before submitting an issue;
- **Description:** In your description, you need to explain in detail what happened, as you would explain to someone that doesn't know anything about your website or Elementor. Give as much information as possible. This is important information that will help solve your issue quicker;
- **Steps to replicate:** Think of it as explaining to someone over the phone how to replicate your issue. They can't see the problem, so explain it in detail in a way that we can try to replicate your issue in our own environments. Be detailed;
- **Isolating the Problem:** You need to make sure this only happens while using the latest versions of Elementor, Elementor Pro, and the Hello Elementor theme. We do not provide support for 3rd-party themes and plugins.
- **System info:** Please include the [System Info](https://elementor.com/help/systems-info/) report as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml) available in your _WordPress Dashboard > Elementor > System Info_. This information is crucial for us to create an exact copy of your environment and validate your issue. Watch [this video](https://www.youtube.com/watch?v=tDXdynsnYuA) to learn how to download the report. We can't accept issues without it.
- **One issue per thread:** We can't track multiple issues per thread. Open a new issue for each subject.

Without them, we simply can't help. 😕 

👉 Please follow the guidelines for opening a new [🐞 Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml), so we can help you better.


### ⚠️ Note that GitHub issues are reserved for Elementor 🐞 Bug Reports only, and should not be used for support.

❔ If you have questions or need general support, please use one of the following channels:

- [Elementor Global Community](https://facebook.com/groups/Elementors/) on Facebook
- [Elementor Help Center](https://docs.elementor.com)
- [Support Forum](https://wordpress.org/support/plugin/elementor) on WordPress.org 
- [Elementor Developers Documentation](https://developers.elementor.com/)

💎 If you have purchased [Elementor Pro](https://elementor.com/pro), you are entitled to personal support, please see your purchase email or visit your [Elementor Account page](https://my.elementor.com/) for details.


#### For  💡 **New Ideas** or 🚀 **Feature Requests** please [create a new Discussion](https://github.com/elementor/elementor/discussions/new) in the [Discussions](https://github.com/elementor/elementor/discussions) tab.

Best regards
